backend/business/core/collectors/github/fetchers/repos_info_fetcher.go

```diff
@@ -124,6 +131,7 @@ func (f *ReposInfoFetcher) buildResult(data []repoInfo) *map[string]RepoInfoResu
 			ForksCount:              int(edge.Node.Forks.TotalCount),
 			IssuesCount:             int(edge.Node.Issues.TotalCount),
 			OpenIssuesCount:         int(edge.Node.OpenIssues.TotalCount),
+			OpenIssuesCreatedAt:     time.Time(edge.Node.OpenIssues.Nodes[len(edge.Node.OpenIssues.Nodes)-1].Issue.CreatedAt.Time),
```

Comment:
This present an interesting problem to our set in that we should probably calculate the median/average for this timestamp but to do that effectively we'll need to rewrite the fetcher above to paginate results for us. @ipc103 and I came up with this function as a starting point


```go
func (f *ReposInfoFetcher) buildResult(data []repoInfo) *map[string]RepoInfoResult {
	holder := make(map[string]RepoInfoResult)

	// find the median age of open issues
	
	for _, edge := range data {
		var medianCreatedTime time.Time
		medianIssueIndex := len(edge.Node.OpenIssues.Nodes) / 2
		if medianIssueIndex > 0 {
			medianCreatedTime = time.Time(edge.Node.OpenIssues.Nodes[medianIssueIndex].CreatedAt.Time)
		}


		holder[string(edge.Node.NameWithOwner)] = RepoInfoResult{
			RepoName:                string(edge.Node.NameWithOwner),
			CollaboratorsCount:      int(edge.Node.Collaborators.TotalCount),
			ProjectsCount:           int(edge.Node.Projects.TotalCount + edge.Node.ProjectsV2.TotalCount),
			DiscussionsCount:        int(edge.Node.Discussions.TotalCount),
			ForksCount:              int(edge.Node.Forks.TotalCount),
			IssuesCount:             int(edge.Node.Issues.TotalCount),
			OpenIssuesCount:         int(edge.Node.OpenIssues.TotalCount),
			OpenIssuesCreatedAt:     medianCreatedTime,
			ClosedIssuesCount:       int(edge.Node.ClosedIssues.TotalCount),
			OpenPullRequestsCount:   int(edge.Node.OpenPullRequests.TotalCount),
			MergedPullRequestsCount: int(edge.Node.MergedPullRequests.TotalCount),
			LicenseName:             string(edge.Node.LicenseInfo.Name),
			WatchersCount:           int(edge.Node.Watchers.TotalCount),
		}
	}

	return &holder
}
```